projects/scss-foundation/docs/absolute-positioning.md

```diff
@@ -13,10 +13,7 @@ Alignment utilities for absolute positioning of content.
 All arguments are optional and default to 0. Sets following properties:
 ```
 	position: absolute;
-	top: $top;
-	right: $right;
-	bottom: $bottom;
-	left: $left;
+	inset: $top, $right, $bottom, $left;
```

Comment:
nice :D

Reply:
@typhoon41 i was just checking inset, values are not separated by coma https://developer.mozilla.org/en-US/docs/Web/CSS/inset

Reply:
Since this is already merged, I created ticket for sorting out docs and this must also be taken into account.

Reply:
@typhoon41 i've just done a research on inset.
values are not separated by comma and values have to be same tim (px, em, whatever but same). so inset: unset 0px 0px 0px won't work

Reply:
Please provide link for that claim. It's illogical that unset or any other property won't work.

Reply:
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/126865855/229733134-b13570b4-3278-4ed1-aa9b-c42478e2fd1f.png)
 situation on NHL